script.js

```diff
@@ -0,0 +1,150 @@
+const mobileOpen = document.getElementById('open-menu');
+const mobileShow = document.getElementById('navbar-item-mobile');
+const mobileClose = document.getElementById('close-menu');
+const popupOpen = document.querySelectorAll('.card-btn');
+const inputName = document.getElementById('user_name');
+const inputEmail = document.getElementById('user_email');
+const sumbitMessage = document.getElementById('sumbit-action');
+const errorMessage = document.getElementById('error');
+
+mobileOpen.addEventListener('click', () => {
+  if (mobileShow.style.display === 'block') {
+    mobileShow.style.display = 'none';
+  } else {
+    mobileShow.style.display = 'block';
+  }
+});
+
+mobileClose.addEventListener('click', () => {
+  if (mobileShow.style.display === 'none') {
+    mobileShow.style.display = 'block';
+  } else {
+    mobileShow.style.display = 'none';
+  }
+});
```

Comment:
- [ ] For some reason, no changes seem to have been made to address [this requested change](https://github.com/sadaf-Daneshgar/First-Project-Microverse/pull/2#discussion_r1223347458). Kindly refer to it and address it.

Reply:
ok now I updated can check please